docs/docs/about.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: About
+permalink: /about
+nav_order: 2
+---
+
+## About OCI
+
+The Open Container Initiative was established in June 2015 by 
+[Docker](https://en.wikipedia.org/wiki/Docker_(software)), one of the many
+early leaders of the container industry that provided an intuitive API to interact
+with [linux containers](https://en.wikipedia.org/wiki/Container_(virtualization)).
+
+### Challenges for OCI
+
+Why did the OCI get established in the first place? This is a longer, more complicated
+story that will be updated by maintainers who can recollect their experience. Generally,
+we can guess that there were several questions and challenges that warranted community
+discussion and support:
+
+ 1. How do I interact with a container, for example, to run or stop it?
+ 2. How do I store contents and metadata inside of a container? 
+ 3. What are the minimal set of required metadata?
```

Comment:
s/are/is/ ?

Reply:
(the english language is not my strong suit)

Reply:
I think metadata is plural https://www.chronicle.com/forums/index.php?topic=31914.0 but "is/are" here might be describing "set" which would be a singular thing, so I think you are right. English is hard!

Reply:
Okay, s/are/is is done!